README.md

```diff
@@ -65,7 +65,7 @@ All commands are run from the root of the project, from a terminal:
 
 DDev already has all the dependencies included.
```

Comment:
Not sure where “DDev” came from, but it [should always be referred to](https://ddev.readthedocs.io/en/stable/developers/writing-style-guide/#writing-style) as DDEV the product or `ddev` the command.

```suggestion
DDEV already has all the dependencies included.
```